lib/components/serverside-data/Table.vue

```diff
@@ -62,12 +71,15 @@ export default {
   },
   computed: {
     tableHeaders() {
-      const headers = this.$attrs.headers.map((h) => ({ ...h, sortable: h.sortable ? h.sortable : false }))
+      const headers = get(this.$attrs, 'headers', []).map((h) => ({ ...h, sortable: h.sortable ? h.sortable : false }))
```

Comment:
Just a thought: How about a computed
```js
headers() {
  return get(this.$attrs, 'headers', [])
}
```
Then you could use `this.headers.map(...)` here and `:colspan="headers.length"` in the `<td>` above. Also, you could then do `v-for="header in headers"` in the other `<td>` though the `v-for="header in $attrs.header"` seems to work even if `$attrs.headers` is undefined 🤔

Reply:
Thank you very much. No idea what I was thinking here 😄